3.Tests/UI/StepDefinitions/Settings/Profile/UpdateEmailStepDefinition.cs

```diff
@@ -0,0 +1,84 @@
+﻿using SeleniumExtras.WaitHelpers;
+using TechTalk.SpecFlow;
+using Todoly.Core.UIElements.Drivers;
+using Todoly.Tests.UI.Steps.Commons;
+using Todoly.Views.WebAppPages;
+
+namespace SeleniumTest.Tests;
+
+[Binding]
+[Scope(Feature = "Email update")]
+public class UpdateEmailStepDefinition : CommonSteps
+{
+    private readonly HomePage _homePage;
+    private LoginPage? _loginPage;
+    private ProfilePage? _profilePage;
+    private readonly ScenarioContext _scenarioContext;
+    public UpdateEmailStepDefinition(ScenarioContext scenarioContext) : base(scenarioContext)
+    {
+        _homePage = new HomePage();
+        _scenarioContext = scenarioContext;
+    }
+
+    [Given(@"the user clicks on the Settings option on the Nav Bar")]
+    public void GiventheuserclicksontheSettingsoptionontheNavBar()
```

Comment:
We need to start implementing more reusable steps, we can create new steps for every new feature. Also step implementation methods should have more concise and clear names.